<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GithubRepoSelector</class>
 <widget class="QDialog" name="GithubRepoSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stk">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageUsername">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLineEdit" name="txtUser"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Github user name / organization:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRepo">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPromptRepo">
         <property name="text">
          <string>Repositories for user %s</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cbxRepos">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="text">
        <string>◀ back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>next ▶</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>GithubRepoSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
